src/quantum/HISTORY.rst

```diff
@@ -3,6 +3,12 @@
 Release History
 ===============
 
+0.18.0
+++++++
+* [2023-02-07] Version intended to work with QDK version 0.27.253010 and Azure CLI 2.41.0 or greater.
+* You can now submit QIR, QIO, and pass-through jobs using the CLI.
```

Comment:
QIO is not fully supported. let's not mention that in the note

Reply:
Revised the QIO support note

Reply:
Deleted mention of QIO